src/Components/Users.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import DeleteIcon from '@mui/icons-material/Delete';
+import SortByAlphaIcon from '@mui/icons-material/SortByAlpha';
+
+export class Users extends Component {
+  render() {
+    const user = this.props.user;
+    return (
+      <div>
+        <table>
+            <tr>
+                <th>id</th>
+                <th>name<button className='sort-btn'><SortByAlphaIcon onClick = {() => this.props.sorting('name')}/></button></th>
+                <th>username<button className='sort-btn'><SortByAlphaIcon onClick = {() => this.props.sorting('username')}/></button></th>
+                <th>email<button className='sort-btn'><SortByAlphaIcon onClick = {() => this.props.sorting('email')}/></button></th>
+                <th>Delete</th>
+            </tr>
+            {user.map(i=>
```

Comment:
also read about map second argument a unique key, and how it makes list rendring efficient in react